### PR TITLE
feat: add sublime-package schema

### PR DESCRIPTION
**Pull request by rchl**
This has been on my disk for quite a while so might need to be re-checked if it's up to date.

Also includes the new `dev_environment` option.

### CODE REVIEW

File: sublime-package.json [hunk 1]
```diff
@@ -0,0 +1,1107 @@
...
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      {
+        "file_patterns": [
+        "/*.sublime-project"
```
Comment (jfcherng) [inline sublime-package.json:1087]: Some indentations are wrong. Let's format this file with LSP-json again?

**Comment by jfcherng** [comment]
From the Pyright docs

> **pyright.disableLanguageServices** [boolean]: Disables all language services except for “hover”. This includes type completion, signature completion, find definition, find references, and find symbols in file. This option is useful if you want to use pyright only as a type checker but want to run another Python language server for langue service features.
> 
> **pyright.disableOrganizeImports** [boolean]: Disables the “Organize Imports” command. This is useful if you are using another extension that provides similar functionality and you don’t want the two extensions to fight each other.

These two are in our settings file but missing in the json schema.

**Comment by jfcherng** [comment]
Yet we have these ST3-compatible in the settings.

image

I vaguely remember there was a port for this to LSP for ST3?

**Comment by rchl** [comment]
> These two are in our settings file but missing in the json schema.

I can see those in the schema.

> I vaguely remember there was a port for this to LSP for ST3?

ST3 still needs `scopes` and `syntaxes`.

**Comment by jfcherng** [comment]
> I can see those in the schema.

Oops. Sorry this is my misedit in my local testing setup.

**Review by jfcherng — APPROVED** [review]

**Pull request merged**